scripts/swap_db.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# backup database
+docker run -it --rm --network sys_default postgres:9.4 pg_dump -h db -U postgres testifi_production > dump.sql
+
+# kill app
+make stop
+
```

Comment:
We also need to `docker volume rm sys_testifi_data` now that the data is stored on a volume

Reply:
Can this happen after we create and fill the new DB?